images/linux/scripts/tests/Node.Tests.ps1

```diff
@@ -0,0 +1,14 @@
+Describe "Node.js" {
+    $binaries = ,"node"
```

Comment:
I think `,` should not be here

Reply:
It must. This creates array of one item

Reply:
why not define array using `$binaries = @("node")`?